bin/elasticsearch.bat

```diff
@@ -33,6 +33,9 @@ FOR /F "usebackq tokens=1* delims= " %%A IN (!params!) DO (
 	)
 )
 
+set HOSTNAME=
+for /f "delims=" %%a in ('hostname') do @set HOSTNAME=%%a
```

Comment:
Can you please set instructions in uppercase to stick with the rest of the file?

Also, I'm not sure this works on older Windows like XP. Do you think you can get it work on it? That would be nice.